foundation/foundation-mda/src/test/java/com/boozallen/aiops/mda/generator/DataRecordsModuleStep.java

```diff
@@ -7,14 +7,11 @@
  * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
  * #L%
  */
-import com.boozallen.aiops.mda.metamodel.element.AbstractModelInstanceSteps;
-import com.boozallen.aiops.mda.metamodel.element.BasePipelineDecorator;
-import com.boozallen.aiops.mda.metamodel.element.Pipeline;
-import com.boozallen.aiops.mda.metamodel.element.RecordElement;
+import com.boozallen.aiops.mda.generator.util.PipelineUtils;
```

Comment:
S: wildcards have some issues (and once you'd experienced them, they leave a mark 😉)

https://rules.sonarsource.com/java/tag/pitfall/RSPEC-2208/

Reply:
This is probably from that one annoying IntelliJ feature I forgot to disable 🙃 I'll take care of it now.